quickwit/quickwit-serve/src/delete_task_api/handler.rs

```diff
@@ -45,8 +45,9 @@ pub struct DeleteQueryRequest {
     /// Query text. The query language is that of tantivy.
     pub query: String,
     // Fields to search on
+    #[serde(rename(deserialize = "search_field"))]
```

Comment:
this is necessary to be consistent with the docs and the search API https://github.com/quickwit-oss/quickwit/blob/ce3ae7769932f3efbe84970a44073966fcc6fd88/quickwit/quickwit-serve/src/search_api/rest_handler.rs#L190-L197